### DISCUSSION TITLE
Transfer messages from Zoom chat to a Mattermost channel after the call ends

### DISCUSSION THREAD

**Discussion by aedott** [mattermost/community#28] [General Ideas]
Often times messages and files are shared during a call, but those messages are "lost" after the call ends (sure they can be shared by the meeting host, but even in that case they're siloed away from any previous or continued conversations in Mattermost. 

I'd love a tool similar to Wrangler that automatically ports messages from Zoom chat into a Mattermost channel after the call ends.

**Comment by esethna** [comment]
That would be great, or even just a way to access the chat log. We have some WIP designs here for zoom improvements: https://www.figma.com/file/qdpsyIqyblj4M4MIXFrHMp/Video-Calling-Designs?node-id=63%3A2324

![image](https://user-images.githubusercontent.com/14333569/122957004-5dc7c500-d336-11eb-866d-1dc9dba45f31.png)

**Reply by jprusch** [reply]
Great idea. I already asked the Zoom marketing guy a while ago, if another chat app (as we use Mattermost) could be integrated instead of their (rather mediocre) solution. They didn't jump on the proposal... But fetching a transcript or the recording (or even generating a link to it) would help a lot!

**Reply by aedott** [reply]
@jprusch I love the idea of just integrating Mattermost chat right into Zoom, too! I figured they wouldn't be super excited about the idea, though.

**Reply by chetanyakan** [reply]
Info card component can also be leveraged to allow seeing zoom chat messages in the RHS

![image](https://user-images.githubusercontent.com/35728906/123300223-ff6a3480-d537-11eb-8c9f-66c2aaf10209.png)
